package.json

```diff
@@ -17,7 +17,10 @@
   "dependencies": {
     "@apollo/datasource-rest": "^4.2.0",
     "@apollo/server": "^4.0.0-alpha.10",
+    "body-parser": "^1.20.2",
     "config": "^3.3.7",
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
```

Comment:
I thought express was already integrated with Apollo Server: https://www.apollographql.com/docs/apollo-server/api/express-middleware.

It is in my package-lock. Do we need to add it explicitly, and also, should we be using the provided Express middleware?
![Screenshot 2023-05-17 at 09 45 49](https://github.com/sul-dlss/folio-graphql/assets/1328900/413eadde-b9d0-47c9-9906-1251dc3813e4)

Reply:
@marlo-longley I don't know. I'm not an expert in this area. If you can tell me what I should do I'll gladly make the change.

Reply:
Following this example seems good to me.